### PR TITLE
Remove Real Name policy

### PR DESCRIPTION
**Pull request by peterbourgon**
So-called "Real Name" policies in online communties are discriminatory toward vulnerable minorities, empirically don't impact nebulous metrics like "civility", and don't belong as part of the GoBridge CoC.

[Quoting the EFF](https://www.eff.org/deeplinks/2011/07/case-pseudonyms),

> There are myriad reasons why individuals may wish to use a name other than the one they were born with. They may be concerned about threats to their lives or livelihoods, or they may risk political or economic retribution. They may wish to prevent discrimination or they may use a name that’s easier to pronounce or spell in a given culture.

Most obviously, female and other minority community members may want to shield their identity for fear of stalking or harassment. But it is of course not limited to minorities: the mailing list has witnessed several messages from Gophers living in nations where access to information is restricted, and who may earnestly fear political or criminal retribution for participating in our communities. _Anyone_ may have legitimate, private reasons for not wanting to expose their real identity in a public forum. Having and enforcing a real name policy is discouraging to those users.

There are other, real reasons people may want or need to avoid using real names online. [Everyday Feminism](http://everydayfeminism.com/2015/09/the-problem-with-real-names/) documents several, including

- Real names may be "dead names" i.e. old names for trans people, who may not be comfortable or able to advertise their transition
- Real names may be reminders of or triggers for trauma

I understand that moderators have suggested in private messages to contributors who don't want to expose their real names that they "make up" normal-sounding first and last names. There are two major problems here. One: this isn't reflected in the on-the-record policy, and so it doesn't help potential members who are discouraged when reading the community rules; also, it creates the potential for discordant judgment from different moderators. Two: the expectation of a normal-sounding first and last name as a pseudonym carries an unfortunate and unnecessary western cultural bias that can itself be discouraging to potential members.

As an amusing, somewhat tangential point of further evidence, real name policies "[may run afoul of legal constraints](https://en.wikipedia.org/wiki/Nymwars#Criticism) such as the German _Telemediengesetz_ federal law, which makes anonymous access to online services a legal requirement".

Finally, I'd like to observe that several high-profile Go community members and contributors have used pseudonyms exclusively during their tenures. Notably, the atom symbol ⚛  (0xe2.0x9a.0x9b) and mhhcbon.

In summary, there is no good reason to have this policy, and many good reasons to be rid of it. Let's be rid of it, please. Thanks for your consideration.

### CODE REVIEW

**Comment by dlsniper** [comment]
The policy does not state that it's mandatory to use the real name in order to participate in online GoBridge spaces.
Probably it should be reworded to have a more clear message about that and the fact that you can use a pseudonym or any other aliases.
However, from the time spent on the Slack community, I can tell you that the worse abusers are people hiding behind non-human names (using just their handle).

**Comment by peterbourgon** [comment]
I would be happy if the wording were changed to

> # Real Names
>
> If you feel comfortable, we encourage you to use your real name in GoBridge online communities. We find it encourages openness and civility in discussions. However, this is not a requirement.

—

> However, from the time spent on the Slack community, I can tell you that the worse abusers are people hiding behind non-human names (using just their handle).

[Hard cases make bad law](https://en.wikipedia.org/wiki/Hard_cases_make_bad_law).

**Comment by bradleyfalzon** [comment]
> However, from the time spent on the Slack community, I can tell you that the worse abusers are people hiding behind non-human names (using just their handle).

I think this is a non-sequitur, any abusers will just choose different, real sounding names.

I agree with removing this policy **and** ensuring the admins don't place pressure on people to use real names if the policy is rephrased.

**Comment by carlisia** [comment]
@peterbourgon I personally value that you try to help us, and we are aware of everything you state. And for this reason, we changed long ago to not require a real name policy. I feel I needed to add this here for all around clarity.

We also no longer have a full name policy. Usually when we currently ping people with a suggestion to use a real sounding name it is because they are behaving in iffy ways to begin with AND have a suspicious name or handle. If this is seen as putting pressure on people, then it might just have to be so to keep the community safe for everyone. 

To reiterate, since we no longer have a real name or a full name policy, the admins do not go around suggesting that people change their name or handle unless there is a good reason. There are plenty of users not using their full names, I'm one of them.

I do like your suggestion to change the wording, thank you  (https://github.com/gobridge/CodeOfConduct/pull/4#issuecomment-292125531).

**Comment by peterbourgon** [comment]
@carlisia Cool! I've updated the PR with the Real Names text I've suggested.

**Comment by peterbourgon** [comment]
@dlsniper 

> The policy does not state that it is mandatory to use the real name in order to participate in online GoBridge spaces.

@carlisia 

> We also no longer have a full name policy. Usually when we currently ping people with a suggestion to use a real sounding name it is because they are behaving in

This does not appear to be true; both @ardan-bkennedy and @flexd appear to be enforcing it as mandatory. [From a discussion in #general a few days ago](https://gophers.slack.com/archives/C029RQSEE/p1491389296095385):

```
<g_questions> Who is this William Kennedy guy?
<g_questions> He keeps nagging me about my name
<g_questions> I changed it from gopher go already and he keeps telling me to
  change [it]
<g_questions> He said something about my name has to be like a social media
  account
<g_questions> I don't know why he just won't let me pick the name I want
<g_questions> . . . the entire issue seems so absurd that I wanted to ask first
<bradleyfalzon> but there's no real name requirement here though is there?
  https://github.com/gobridge/CodeOfConduct/blob/master/README.md#name-policy ?
  it's encouraged, but not required ?
<flexd> @bradleyfalzon It's encouraged.. Like you have a nick/handle and a name
  that at least sounds like it could be a name.. I don't know if it is, but it
  does not matter because you are clearly identified by it.
<g_questions> What's the point of such a rule? You don't even know my real name
<g_questions> Someone could use a fake name anyway
<flexd> Yes, and that is fine, as long as it's an actual name
<flexd> @g_questions If you went to a BMW store and was going to ask them
  questions about cars, would you say your name was "BMW questions" when
  the seller asked? That's all we are asking here.
<g_questions> So can an @admin tell me if I am required to change it?
<flexd> Yes, please.
```

**Comment by flexd** [comment]
@peterbourgon For what it's worth, I agree with what you say, and the change you suggested is good. As an admin I do not have to agree with all the rules, but it is my job to enforce them and keep us civilized.  In that specific case, the rule was enforced because the user had the name "go golang" and the handle @go_questions, which to me does not seem like a person, and might be confused to be either a official account or a bot.
